packages/@aws-cdk/aws-dynamodb/lib/table.ts

```diff
@@ -474,13 +475,28 @@ export class Table extends Resource {
   }
 
   /**
-   * Permis an IAM principal all stream data read operations for this
+   * Permits an IAM Principal to list all DynamoDB Streams.
```

Comment:
Comment is imprecise I think. Should be something like "Permits an IAM Principal to list the stream attached to this DynamoDB table".